parsers/cobol.c

```diff
@@ -132,21 +227,37 @@ static void initializeCobolParser (langType language)
 	cobol = language;
 
 	addLanguageCallbackRegex (cobol,
-					  COBOL_REGEX_PREFIX
-					  "[0-9]+[ \t]+([A-Z0-9][A-Z0-9-]*)[ \t]+("
-					  "BLANK|OCCURS|IS|JUST|PIC|REDEFINES|RENAMES|SIGN|SYNC|USAGE|VALUE"
-					  ")",
-					  "{icase}",
-					  make_tag_for_data_maybe, NULL, &cobol);
+		COBOL_REGEX_PREFIX_FIXED
+		"[0-9]+[ \t]+([A-Z0-9][A-Z0-9_-]*)[ \t]+("
+		"BLANK|OCCURS|IS|JUST|PIC|REDEFINES|RENAMES|SIGN|SYNC|USAGE|VALUE|COMP\\-1|COMP\\-2|COMP\\-3|COMP\\-X|COMP"
+		")",
+		"{icase}",
+		make_tag_for_data_maybe, NULL, &cobol);
+
+	addLanguageCallbackRegex(cobol,
+		COBOL_REGEX_PREFIX_RELAXED
+		"[0-9]+[ \t]+([A-Z0-9][A-Z0-9_-]*)[ \t]+("
+		"BLANK|OCCURS|IS|JUST|PIC|REDEFINES|RENAMES|SIGN|SYNC|USAGE|VALUE|COMP\\-1|COMP\\-2|COMP\\-3|COMP\\-X|COMP"
+		")",
+		"{icase}",
+		make_tag_for_data_maybe, NULL, &cobol);
+
 	addLanguageCallbackRegex (cobol,
-					  COBOL_REGEX_PREFIX
-					  "([A-Z0-9][A-Z0-9-]*)\\.",
-					  "{icase}",
-					  make_tag_for_paragraph_maybe, NULL, &cobol);
+		COBOL_REGEX_PREFIX_FIXED
+		"([A-Z0-9][A-Z0-9-]*)\\.",
+		"{icase}",
+		make_tag_for_paragraph_maybe, NULL, &cobol);
+
+	addLanguageCallbackRegex(cobol,
+		COBOL_REGEX_PREFIX_RELAXED
+		"([A-Z0-9][A-Z0-9-]*)\\.",
+		"{icase}",
+		make_tag_for_paragraph_maybe, NULL, &cobol);
+
 	addLanguageCallbackRegex (cobol,
-					  "^[ \t]*COPY[ \t]+([A-Z0-9][A-Z0-9-]*)\\.",
```

Comment:
If possible, I would like you to follow the indentation style used in the other lines instead of introducing your style.

Doesn't "Edit config" explained in http://docs.ctags.io/en/latest/contributions.html?highlight=editor#c-language help you?